### PR TITLE
Rename .raw to .original fields

### PR DESCRIPTION
**Pull request by ruflin**
* Rename `user_agent.raw` to `user_agent.original` and make it a keyword. I think this always should have been a keyword.
* Rename `event.raw` to `event.original`.

Closes https://github.com/elastic/ecs/issues/102

### CODE REVIEW

File: schemas/user_agent.yml [hunk 1]
```diff
@@ -6,8 +6,8 @@
     The user_agent fields normally come from a browser request. They often
     show up in web service logs coming from the parsed user agent string.
   fields:
-    - name: raw
-      type: text
+    - name: original
+      type: keyword
```
Comment (webmat) [inline schemas/user_agent.yml]: I would actually make this one multi-field.

Full text search will be useful for weird user agents that aren't recognized by the US parsers (various language HTTP libs, obscure crawlers, etc). Still useful to filter for everything that contains "Python", for example. So for this we need to retain full text search.

So I would go with FTS at the top + `.keyword` for the nested field.

Comment (webmat) [reply schemas/user_agent.yml]: Or you can consider this out of scope for now, and wait for the full review of fields in #104 to go MF. In that case we should leave it at `text` indexing.

File: use-cases/filebeat-apache-access.md [hunk 2]
```diff
@@ -21,7 +21,7 @@ ECS fields used in Filebeat for the apache module.
 | <a name="http.response.body_sent.bytes"></a>*http.response.body_sent.bytes*  | *Http response body bytes sent, currently apache.access.body_sent.bytes*  | long  |   | `117`  |
 | <a name="http.referer"></a>*http.referer*  | *Http referrer code, currently apache.access.referrer<br/>NOTE: In the RFC its misspell as referer and has become accepted standard*  | keyword  |   | `http://elastic.co/`  |
 | <a name="user_agent.&ast;"></a>*user_agent.&ast;*  | *User agent fields as in schema. Currently under apache.access.user_agent.*<br/>*  |   |   |   |
-| [user_agent.raw](https://github.com/elastic/ecs#user_agent.raw)  | Raw user agent. Currently apache.access.agent  | text  |   | `http://elastic.co/`  |
+| <a name="user_agent.raw"></a>*user_agent.raw*  | *Raw user agent. Currently apache.access.agent*  | text  |   | `http://elastic.co/`  |
```
Comment (webmat) [inline use-cases/filebeat-apache-access.md]: Ha, good catch. You've looked for `event.raw` in the examples as well, I assume?

File: schemas/user_agent.yml (hunk 1 above)
Comment (ruflin) [reply schemas/user_agent.yml]: I consider it out of scope for now :-)

File: use-cases/filebeat-apache-access.md (hunk 2 above)
Comment (ruflin) [reply use-cases/filebeat-apache-access.md]: you mean the use cases? Hm, I didn't :-(

File: schemas/user_agent.yml (hunk 1 above)
Comment (webmat) [reply schemas/user_agent.yml]: Agreed. So you'll revert back to `text` indexing?

Comment (ruflin) [reply schemas/user_agent.yml]: Can do. So far I was thinking it's a bug that it's text :-)

**Comment by ruflin** [comment]
@webmat New version pushed and use cases updated.

**Pull request merged**